crates/cubecl-core/src/codegen/execution.rs

```diff
@@ -74,12 +74,14 @@ where
     /// Execute a dynamic kernel.
     #[allow(unused)]
     pub fn execute(self, launch: CubeCountSettings<R::Server>) {
-        execute_dynamic::<R, K, f32, f32, f32>(
+        execute_dynamic::<R, K, f32, f32, f32, f32, f32>(
```

Comment:
This is the old way of launching kernels, we will probably delete it once everything is migrated to the newer CubeCL API. What made you change it?

Reply:
I wasn't aware it was outdated - it's still used by at least a few `burn` kernels. I changed it because it includes a match on the element type, and I figured I should at least build the scaffolding to allow atomic scalar params. Tehy need to be treated separately because `wgpu` treats them  as entirely separate types. I can undo it if it's legacy code anways.